### ISSUE TITLE
Can't open Spotify links in their native app when dns is enabled

### ISSUE THREAD

**Issue by iuliangcata** [AdguardTeam/AdguardForAndroid#4995]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

latest

### Environment

```markdown
- OS: Android 13
- Device: Samsung, Pixel, Realme (any?)
- Firmware: Latest to date
```

### Root access

- [ ] Yes, I have it.

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard Spanish/Portuguese filter

### Issue Details

Steps to reproduce:
1. Enable DNS protection
2. Ask someone to send you a Spotify song link
3. Open the link (It opens the installed app but it redirects nowhere, like it's missing info)
4. Now disable DNS protection
5. Click the link again and it will open the app, except it now redirects the app correctly to the song.

### Expected Behavior

Open the Spotify link in the native app and redirect internally to the song. 

### Actual Behavior

Opens the Spotify link in the native app and does nothing, remains on main window of the app, like it's missing part of the information or url.

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

**Comment by maxikuzmin** [comment]
@iuliangcata yes, you are right, this is also recurring for me. The problem is with the AdGuard DNS filter. We will fix it

**Comment by dlibis** [comment]
Is there any update? Its still happens

**Comment by varun4** [comment]
Any update on this?
This is breaking spotify shares, spottily jam sessions -  anything that works on spotify shortlinks.
